.github/workflows/frontend-test.yml

```
@@ -1,42 +1,51 @@
-              run: npm install
-            - name: Test with coverage
-              run: make coverage
-            - name: Coveralls
-              uses: coverallsapp/github-action@v2
-              with:
-                  path-to-lcov: './coverage/lcov.info'
-                  github-token: ${{ secrets.GITHUB_TOKEN }}
+ frontend-test:
+   runs-on: ubuntu-latest
+   steps:
+     - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)